jib-core/src/main/java/com/google/cloud/tools/jib/registry/ErrorResponseUtil.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.jib.registry;
+
+import com.google.api.client.http.HttpResponseException;
+import com.google.cloud.tools.jib.json.JsonTemplateMapper;
+import com.google.cloud.tools.jib.registry.json.ErrorEntryTemplate;
+import com.google.cloud.tools.jib.registry.json.ErrorResponseTemplate;
+import java.io.IOException;
+import java.util.List;
+
+/** Helper class for parsing {@link ErrorResponseTemplate} from JSON. */
+public class ErrorResponseUtil {
+
+  /**
+   * Extract an {@link ErrorCodes} response from an error object.
+   *
+   * @param ex the response exception
```

Comment:
This Javadoc error is causing the build failure. There is no `ex`.

Reply:
Argh!